### ISSUE TITLE
An issue about RELATION model and article

### ISSUE THREAD

**Issue by atabeyunlu** [micahwang/RELATION#5]
Hi,

I was examining the code just now and I am confused about the loss function. In the training script, loss does not seem to be calculated as it was described in the article. After summing all other losses, at Line 238, total loss was assigned to source_latloss then summed with source_simse and fed backward. Am I missing something or misunderstanding the implementation or is it a coding error? In the article, it is stated that latent loss should also be summed. I am leaving the link for the part that troubles me.

https://github.com/micahwang/RELATION/blob/2c80cc152acf32b7b441ea583a5a05a5409deac5/model/train.py#L238

My other question is, if this is a coding error, do published results come from the model including this line of code where other losses are not included? I am asking this because if this is the case, it contradicts the loss defined in the article. I just want to clear out my confusion. Thanks for your time and answer.
